home/migrations/0037_alter_contentpage_whatsapp_body.py

```diff
@@ -76,6 +75,24 @@ class Migration(migrations.Migration):
                                         ),
                                     ),
                                 ),
+                                (
+                                    "example_values",
+                                    wagtail.blocks.ListBlock(
+                                        wagtail.blocks.StructBlock(
+                                            [
+                                                (
+                                                    "example_values",
+                                                    wagtail.blocks.CharBlock(
+                                                        label="Example Value"
+                                                    ),
+                                                )
+                                            ]
+                                        ),
+                                        default=[],
+                                        help_text="Please add example values for all variables used in a WhatsApp template",
+                                        label="Variable Example Values",
+                                    ),
+                                ),
```

Comment:
For this migration, we probably don't want to edit this existing migration, because systems that already have this migration applied won't then do this additional step. What I would do here is revert the changes to this file, and run `makemigrations` again, to generate this as an `0038` migration.

Reply:
Cool thanks

Reply:
I think we still need to remove these changes here

Reply:
Sorry, I discarded the changes in git, but looks like that didn't work.  Tried copying down the contents of the file that was in main now